### PR TITLE
Refactor: agency index

### LINKED ISSUE #367: Update rendering of Agency index page
## Background

In tandem with #366, this ticket moves the bulk of the contextual information from the application index to the agency index.

We need more flexibility on this page than the current viewmodel / page template allows for.

image

## Tasks

* [x] Create/update text entries in English and Spanish for each paragraph, title, etc.
* [x] Can we embed the **bold** font style in the translation?
* [x] Create text entries for a pre-button label / subhead
* [x] Create a new template `core/agency_index.html` to use for this page
* [x] In the view use the new template

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #367 

Follows approach outlined in the ticket. Using a new template specific to `agency_index` gave us the flexibility to use `blocktranslate` for embedding the link to "Learn more about Cal-ITP Benefits."

Note that I didn't attempt to style the page. I think @machikoyasuda has an idea for doing a focused cleanup of styles in a separate ticket. Let me know though if that's not correct! 🙏 

Latest screen from Figma mockup:
image

Current view on this branch:
image

Here's the Before (from dev branch): 
image

### UI tests pass
image

### CODE REVIEW

File: benefits/locale/es/LC_MESSAGES/django.po
Comment (machikoyasuda) [inline benefits/locale/es/LC_MESSAGES/django.po:251]: The Spanish translations are here https://docs.google.com/document/d/1uzGaFF73CKoC7nftYhQ5329obHcz9SJj/edit from @o-ram https://github.com/cal-itp/benefits/issues/206#issuecomment-1087992783

image

File: benefits/locale/en/LC_MESSAGES/django.po
Comment (machikoyasuda) [inline benefits/locale/en/LC_MESSAGES/django.po:293]: Since this was changed, the specs that use `Let’s do it!` need to be updated to `Get started`

Comment (angela-tran) [reply benefits/locale/en/LC_MESSAGES/django.po:293]: Thanks! Gets me every time

I fixed this in 474b5b7

File: benefits/locale/es/LC_MESSAGES/django.po
Comment (machikoyasuda) [reply benefits/locale/es/LC_MESSAGES/django.po:251]: Oh, never mind! Just realized this copy is slightly different.

**Review by machikoyasuda — APPROVED** [review]
LGTM! After merging this PR, need to make a ticket to: 

1. Get all new Spanish translations and add them
2. Finalize CSS for all pages *once* all the copy and new pages/elements are in.

File: benefits/locale/es/LC_MESSAGES/django.po
Comment (machikoyasuda) [reply benefits/locale/es/LC_MESSAGES/django.po:251]: TODOs are fine here

**Comment by angela-tran** [comment]
I'll wait until tomorrow to merge this, in case anyone else wants to review

**Review by thekaveman — APPROVED** [review]
Nice!

**Comment by angela-tran** [comment]
> LGTM! After merging this PR, need to make a ticket to:
> 
>     1. Get all new Spanish translations and add them
> 
>     2. Finalize CSS for all pages _once_ all the copy and new pages/elements are in.

Thanks for this list @machikoyasuda ! I think https://github.com/cal-itp/benefits/issues/359 captures that we're waiting to get finalized Spanish translations. I wouldn't be opposed to also having our own dev-centric ticket for actually adding those Spanish translations in if you think that would be good to have.

Do you want to create the GitHub issue for finalizing the CSS once all the pages/elements are in?

**Comment by machikoyasuda** [comment]
@angela-tran Sounds good! Could you create that CSS ticket after you merge this PR?? Thank you 🙏

**Comment by thekaveman** [comment]
@angela-tran when you create that new ticket, can you please add it to the list on #363 too?

**Pull request merged**
